app.js

```diff
@@ -2,7 +2,8 @@ var Twit = require('twit');
 var path = require('path');
 var slackbot = require('node-slackbot');
 require('dotenv').config({
-	path: path.resolve('./env_variables')
+	path: path.resolve('./env_variables'),
+	silent: true
```

Comment:
Production Node environments should have `NODE_ENV="production"`. If EB doesn't do this already with a default Node setup, we should do it. And once that's there, you can just do this:
```
silent: process.env.NODE_ENV === 'production'
```